cloud-optimized-geotiffs/cogs-overview_resampling.ipynb

```diff
@@ -1,19 +1,20 @@
 {
```

Comment:
<p>removing <code>over</code> changes the meaning, perhaps <code>over-represents</code> is the correct term </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Fixed!